src/css/components/po-tag/po-tag.css

```diff
@@ -123,6 +123,10 @@
 }
 
 .po-tag-label {
+  height: 100%;
+  display: inline-flex;
+  align-items: center;
+
```

Comment:
width: 100%;
justify-content: center;

para quando a orientação for horizontal o label ficar centralizado junto com a tag